integration/src/main/java/scripts/api/TdrDatasetsApi.java

```diff
@@ -0,0 +1,166 @@
+package scripts.api;
+
+import bio.terra.datarepo.api.ProfilesApi;
+import bio.terra.datarepo.client.ApiException;
+import bio.terra.datarepo.model.ColumnModel;
+import bio.terra.datarepo.model.DatasetModel;
+import bio.terra.datarepo.model.DatasetRequestModel;
+import bio.terra.datarepo.model.DatasetSpecificationModel;
+import bio.terra.datarepo.model.IngestRequestModel;
+import bio.terra.datarepo.model.PolicyMemberRequest;
+import bio.terra.datarepo.model.TableDataType;
+import bio.terra.datarepo.model.TableModel;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.client.DatarepoClient;
+
+public class TdrDatasetsApi {
```

Comment:
why not extend datasetsapi?

Reply:
Because I want to limit the API exposed to the caller. I don't want to allow a caller to call `createDataset()` or `deleteDataset()` directly, for example.